WPFWijkagent/WPFWijkagent/MainWindow.xaml.cs

```diff
@@ -17,6 +18,9 @@ public partial class MainWindow : Window
         // controls the offences for this window
         private readonly OffenceController _offenceController = new OffenceController();
         private bool _addModeActivated = false;
+        SocialMediaDialogue social;
+        private List<Offence> offenceList = new List<Offence>();
```

Comment:
offenceList aanpassen naar _offenceList (staat in programmeerstandaarden)

Reply:
aangepast

Reply:
aangepast